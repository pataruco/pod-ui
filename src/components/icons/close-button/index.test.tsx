import { shallow } from 'enzyme';
import React from 'react';
import CloseButton from './index';

describe('CloseButton', () => {
  it('renders', () => {
    expect(shallow(<CloseButton />)).toMatchSnapshot();
  });
});
