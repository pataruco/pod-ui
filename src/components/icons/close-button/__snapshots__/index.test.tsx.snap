// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseButton renders 1`] = `
<svg
  className="svg"
  viewBox="0 0 52 52"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    className="circle"
    cx="26"
    cy="26"
    r="24"
  />
  <line
    className="lines"
    x1="15"
    x2="37"
    y1="15"
    y2="37"
  />
  <line
    className="lines"
    x1="15"
    x2="37"
    y1="37"
    y2="15"
  />
</svg>
`;
